CONTRIBUTING.md

```diff
@@ -1 +1,8 @@
-We require contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+# Contribution guidelines
+
+We ***require*** contributors to sign our Contributor License Agreement. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla. If you have any questions, please drop us a line at cla@fb.com. Thanks!
+
+## Submitting Pull-Requests
+* Please make sure it builds : `make -C infer clean && make -C infer <target>`, refer to installation document for details
+* Add a test for your change, and run the all the tests: `./scripts/test.sh`
```

Comment:
Maybe qualify this a bit, since not all changes require a new test: `If relevant, add a test ...`?

Also, could you include the location of the tests so people know where to look? infer/tests/codetoanalyze/<language>/ and infer/tests/{endtoend,frontend}/<language>/ come to mind.